### PR TITLE
add tiny js world by jsmuse

### PR DESCRIPTION
**Pull request by jsmuse**
### add tiny js world by jsmuse

[Demo](https://jsmuse.github.io/a-tiny-JS-world/) |
[Code base](https://github.com/jsmuse/a-tiny-JS-world)

The code is submitted in a dedicated feature branch.
Please, review.

### CODE REVIEW

File: submissions/jsmuse/a-tiny-JS-world/index.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    gender: 'female',
+    legs: 2,
+    hands: 2,
+    saying: catSay,
+};
+
+const listObj = (obj) => {
+    let str = '';
+
+    for (item in obj) {
```
Comment (OleksiyRudenko) [inline submissions/jsmuse/a-tiny-JS-world/index.js]: The order of properties in an object is not guaranteed. Please, list `obj` properties explicitly and also employ `Array.join`.

File: submissions/jsmuse/a-tiny-JS-world/index.js [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    }
+
+    return str;
+};
+
+print(listObj(dog));
+print(listObj(cat));
+print(listObj(woman));
+print(listObj(man));
+print(listObj(catWoman));
```
Comment (OleksiyRudenko) [inline submissions/jsmuse/a-tiny-JS-world/index.js]: There is a pattern above.
Hint:
`[varA, varB].forEach(item => whateverFunction(whateEverAnotherFunction(item)));`

Comment (OleksiyRudenko) [inline submissions/jsmuse/a-tiny-JS-world/index.js]: Please, also ensure your code files end with an empty line. It is important. You can tune your IDE to do this dirty job for you.
Not required to fix right now.

File: submissions/jsmuse/a-tiny-JS-world/index.js (hunk 1 above)
Comment (jsmuse) [reply submissions/jsmuse/a-tiny-JS-world/index.js]: This is made not for order. Zero is needed to hide keys with properties 0. I check if property not equal zero and show it.

**Comment by jsmuse** [comment]
@OleksiyRudenko thanks for review. Please check again :)

File: submissions/jsmuse/a-tiny-JS-world/index.js (hunk 1 above)
Comment (OleksiyRudenko) [reply submissions/jsmuse/a-tiny-JS-world/index.js]: @jsmuse `for in`, `Object.keys` and other `Object` methods iterate or list object properties in some arbitrary order, there is no guarantee that e.g. `species` will come first, and `name` will follow `species`.
But we have to print properties in a very particular order. This means that we need to list properties explicitly.

To make the problem more straightforward imagine that you need to print all properties with an exception of `legs`.
How would you solve this task?
Please, suggest a fragment of code, i.e. `listObj` function.

Comment (OleksiyRudenko) [reply submissions/jsmuse/a-tiny-JS-world/index.js]: @jsmuse A hint:
```
[
  varA,
  varB,
  obj.propA,
].join('; ');
```

**Comment by jsmuse** [comment]
@OleksiyRudenko thank you for explain, I'am already fix this.

**Pull request merged**